pydeface/__main__.py

```diff
@@ -74,78 +102,39 @@ def main():
         "--verbose", action='store_true',
         help="Show additional status prints. Off by default.")
 
+    parser.add_argument('--dbg', action='store_true', dest='debug',
```

Comment:
I won't be the final one to test and approve, but I can give general feedback here. I would suggest having the argument as `--debug`, it's more intuitive than `dbg` which, if I *had* to guess I'd start to think debug, but it could be something random that begins with database, etc. Since this is meant for a human to read, let's give them that.